src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/readthreading/ReadThreadingAssembler.java

```diff
@@ -258,29 +258,35 @@ private static boolean pathIsTooDivergentFromReference(final Cigar c) {
      */
     private void printDebugGraphTransform(final BaseGraph<?,?> graph, final String fileName) {
         File outputFile = new File(debugGraphOutputPath, fileName);
-        if ( debugGraphTransformations ) {
-            if ( PRINT_FULL_GRAPH_FOR_DEBUGGING ) {
-                graph.printGraph(outputFile, pruneFactor);
-            } else {
-                graph.subsetToRefSource(10).printGraph(outputFile, pruneFactor);
-            }
+        if ( PRINT_FULL_GRAPH_FOR_DEBUGGING ) {
```

Comment:
Since you removed the `if ( debugGraphTransformations ) {` guard from this method, did you check all usages of it to ensure that the callsites are all now checking the `debugGraphTransformations` flag?

Reply:
Yes. Every usage. I will add a comment to be doubly sure that its someone elses fault if that causes problems in the future.